inc/functions/admin.php

```diff
@@ -39,7 +39,7 @@ function rocket_need_api_key() {
  */
 function rocket_renew_all_boxes( $uid = null, $keep_this = [] ) {
 	// Delete a user meta for 1 user or all at a time.
-	delete_metadata( 'user', $uid, 'rocket_boxes', null === $uid );
+	delete_metadata( 'user', $uid, 'rocket_boxes', '', null === $uid );
```

Comment:
```suggestion
	delete_metadata( 'user', $uid, 'rocket_boxes', '', ! $uid );
```

because we are calling it with zero value here:
https://github.com/wp-media/wp-rocket/blob/d89551a8af8d21e90b74861f97f5317b326c957d/inc/admin/upgrader.php#L28

What do u think?

Reply:
@engahmeds3ed Can you push that change on your own?

Reply:
@engahmeds3ed Is it possible to push that on our own? 🙏